examples/upbound-aws-provider/composite-resources/serverless-examples/microservice/README.md

```diff
@@ -0,0 +1,356 @@
+container.lambda.aws.upbound.awsblueprints.io         XLambdaFunction   awsblueprints.io/v1alpha1   23m
+lambda-authorizer-invoke.iampolicy.awsblueprints.io   IAMPolicy         awsblueprints.io/v1alpha1   23m
+lambda-invoke-apigw.iampolicy.awsblueprints.io        IAMPolicy         awsblueprints.io/v1alpha1   23m
+lambda-invoke.iampolicy.awsblueprints.io              IAMPolicy         awsblueprints.io/v1alpha1   23m
+microservices.upbound.awsblueprints.io                XMicroservice     awsblueprints.io/v1alpha1   23m
+read-kms.iampolicy.awsblueprints.io                   IAMPolicy         awsblueprints.io/v1alpha1   23m
+read-s3.iampolicy.awsblueprints.io                    IAMPolicy         awsblueprints.io/v1alpha1   23m
+read-sqs.iampolicy.awsblueprints.io                   IAMPolicy         awsblueprints.io/v1alpha1   23m
+rest.apigateway.aws.upbound.awsblueprints.io          XApiGateway       awsblueprints.io/v1alpha1   23m
+s3.lambda.aws.upbound.awsblueprints.io                XLambdaFunction   awsblueprints.io/v1alpha1   23m
+sns.notification.upbound.awsblueprints.io             XNotification     awsblueprints.io/v1alpha1   23m
+table.dynamodb.awsblueprints.io                       XDynamoDBTable    awsblueprints.io/v1alpha1   23m
+write-cloudwatch-metrics.iampolicy.awsblueprints.io   IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-dynamodb.iampolicy.awsblueprints.io             IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-firehose-s3.iampolicy.awsblueprints.io          IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-firehose.iampolicy.awsblueprints.io             IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-s3.iampolicy.awsblueprints.io                   IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-sqs.iampolicy.awsblueprints.io                  IAMPolicy         awsblueprints.io/v1alpha1   23m
+```
+
+### Build Lambda function packages
+
+This example uses ZIP packaging of the Lambda functions. 
+ - Navigate to `./src/logic/` directory
+ - Create ZIP file `microservice-business-logic.zip` with the `lambda_function.py` in it:
+    ```shell
+    zip microservice-business-logic.zip lambda_function.py
+    ```
+ - Navigate to `./src/authorizer/` directory
+ - Create ZIP file `microservice-authorizer.zip` with the `lambda_function.py` in it:
+    ```shell
+    zip microservice-authorizer.zip lambda_function.py
+    ```
+ - Create Amazon S3 bucket that will be used to store Lambda ZIP packages (make sure to use unique bucket name):
+   ```shell
+   aws s3api create-bucket --bucket <my-zip-packaging-bucket-name> --region <replace-with-aws-region> # example `us-east-1`
+   ```
+ - Upload both ZIP files to the S3 bucket you just created:
+   ```shell
+   aws s3 cp microservice-business-logic.zip s3://<my-zip-packaging-bucket-name>/ 
+   aws s3 cp microservice-authorizer.zip s3://<my-zip-packaging-bucket-name>/ 
+   ```
```

Comment:
Could this be a shell script, like [this one](https://github.com/awslabs/crossplane-on-eks/blob/main/examples/upbound-aws-provider/composite-resources/serverless-examples/object-processor-app/build-and-upload-zip.sh)?

Also, s3api errors out if you try to use a region different that us-east-1 with
```An error occurred (IllegalLocationConstraintException) when calling the CreateBucket operation: The unspecified location constraint is incompatible for the region specific endpoint this request was sent to.```

I had to add `--create-bucket-configuration LocationConstraint=us-east-2` for it to work. Alternative would be to use `mb` instead of `s3api`.

Reply:
Updated using script referred